packages/bierzo-wallet/src/store/extension/actions.ts

```diff
@@ -4,6 +4,6 @@ export const setExtensionStateAction = (
   connected: boolean,
   installed: boolean,
 ): SetExtensionStateActionType => ({
-  type: '@@extension/SET_EXTENSION_STATE',
```

Comment:
My idea was to use prefix to describe action relation, so `@@extension/SET_EXTENSION_STATE` means that this action relates to `extensions` state. I think that `EXTENSION part in the `SET_EXTENSION_STATE` name is redundant in this case and it is better to call this action something like `@@extension/SET_STATE`.

Reply:
//cc @abefernan 

I see, normally I use redux to describe the use storage of the whole app, that is why I use @@bw (bierzo-wallet).

I totally understand your point and to be honest I do not mind to adapt myself to your way of doing it. 

I need to create a final PR where I call this action after login. In that PR I will update the code based on your remarks. So feel free to approve this PR. :wink: